nservicebus/messaging/delayed-delivery.md

```diff
@@ -43,11 +43,17 @@ snippet:delayed-delivery-datetime
 
 ## Caveats
 
-In order to use message deferral the Timeout Manager feature must be enabled. When deferring a message, it is sent to the timeout manager requesting it to deliver the message at a later time.
+Delayed delivery of messages is only supported when the following requirements are met:
+
+* The Timeout Manager feature is enabled or
```

Comment:
What about this [case](https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs#L23)? I guess that falls under `TM is enabled`.